react-minesweeper/components/board.jsx

```diff
@@ -15,7 +15,7 @@ class Board extends React.Component{
       <div id="board">
         <div>WHATS UP!!!!!!!!!!!!!!!!!</div>
         <div>WHAT IS LEFT?!!!!!!!!!!!!!!!!!</div>
-
+        <div>What is right???</div>
```

Comment:
let's change this to 'What is down?' instead

Reply:
@joshling1919 wording has been updated